mysite/templates/index.html

```diff
@@ -24,18 +24,35 @@
     
   }
 
+  .imge {
```

Comment:
I think its the right time to change class name from `imge` to something readable as there are only 5 usages within same file.
Lets change it .